### ISSUE TITLE
can't assign worker after moving hutblock

### ISSUE THREAD

**Issue by SivartMcDorf** [ldtteam/minecolonies#8496]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest MineColonies Version?

- [X] I am running the latest **alpha** version of MineColonies for my Minecraft version.
I am also running the latest (possible) versions of other mods that are part of my problem.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies/Structurize [Wiki](https://wiki.minecolonies.ldtteam.com/) or I asked on [discord](https://discord.minecolonies.com/). Or I was sent from discord to open an issue here.

### Minecraft Version

1.18

### MineColonies Version

867A

### Structurize Version

408A

### Current Behavior

can't assign builder to builder hut after moving it, it is not available until after a server reboot.

### Expected Behavior

place down builder hut then add a worker

### Reproduction Steps

disassemble builder hut
pick it up
place it down

### Logs

https://gist.github.com/SivartMcDorf/3f2eccd404fc9d4a1045511f0307b71b

### Footer

---
## Viewers
* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Talyda** [comment]
Just adding to this, I had this same thing happen on the exploration server - but I wasn't sure if a server reboot had happened to fix it. After restarting game several times, and getting frustrated I just logged out for a while. When I logged back in, I could assign them again. So with McDorf experiencing it too, I am guessing the server rebooted while I was off.  Hadn't seen anyone else mention it until yesterday, so I didn't report it, cos I thought it was a fluke one off thing
